.CI/Test/FileSystem.c

```diff
@@ -4,8 +4,23 @@
 #include <assert.h>
 #include <string.h>
 
+typedef enum {
```

Comment:
Wouldn't it have made more sense to move the other ModelicaFileType declaration from ModelicaInternal.c to ModelicaInternal.h and re-use that?

Basically, I don't like such code duplication.